lib/models/project.js

```diff
@@ -104,6 +104,10 @@ Project.prototype.hasDependencies = function() {
  */
 Project.prototype.setupNodeModulesPath = function() {
   this.nodeModulesPath = nodeModulesPath(this.root);
+
+  this.emberCliNodeModulesPath = this.nodeModulesPath &&
```

Comment:
ya, NPM and move stuff around. Our only basic guarantee is to use require.resolve